runbotics/runbotics-orchestrator-ui/src/src-app/utils/findVariablesInActivities.ts

```diff
@@ -0,0 +1,74 @@
+import {
+    BPMNElement,
+    CamundaInputParameter,
+    CamundaOutputParameter,
+    CamundaParameter,
+    IBpmnGateway,
+    IBpmnSubProcessBusinessObject,
+    ISequenceFlowBusinessObject,
+    getParameterValue
+} from '#src-app/views/process/ProcessBuildView/Modeler/helpers/elementParameters';
+
+export const findVariablesInGateway = (businessObject: IBpmnGateway, searchPhrase: string): boolean => {
+    const gatewayOutgoingFlows: ISequenceFlowBusinessObject[] = businessObject.outgoing as unknown as ISequenceFlowBusinessObject[];
```

Comment:
Can you check once again if those types are ok? I think there is mistake. I think that if `businessObject` is of type IBpmnGateway, `outgoing` should be of type `IBpmnConnection[]`.